### ISSUE TITLE
Request - Email on alert based on regex of alert name?

### ISSUE THREAD

**Issue by Sc0th** [jasonish/evebox#45]
Would be nice :)

**Comment by jasonish** [comment]
I like the idea, of email alerts in general. But it would come after some general "business logic rules" required I think.  Need to be careful to avoid spamming the recipient as well. But in general, yeah..  Not adding as a milestone yet as somethings need to be done first.

**Comment by sgabe** [comment]
This and #126 could be implemented by resolving #82.

**Comment by sgabe** [comment]
@jasonish, do you have any ideas how would you like to approach this topic? I might have some free capacity to try cook something up.

**Comment by jasonish** [comment]
Sorry, I don't. I'm closing this as I find it out of scope for Evebox at this time.
